include/SugarObjects/templates/basic/Basic.php

```diff
@@ -113,9 +113,9 @@ public function getOptInStatusFromSugarField($emailField)
 
             if ($emailAddress->confirm_opt_in == '1') {
                 return 'OPT_IN_PENDING_EMAIL_CONFIRMED';
-            } elseif(!empty($emailAddress->opt_in_email_created)) {
+            } elseif(!empty($emailAddress->confirm_opt_in_sent_date)) {
```

Comment:
Perhaps I could be clearer here. I am concerned because you have renamed and moved the class. You see my pull request expects the ConfirmedOptIn. So I am worried that this is going to clash.

Reply:
Many times I also have to change my code to make it compatible with the repository: for e.g. last time: https://github.com/salesagility/SuiteCRM/pull/4932/files#diff-25e9e5d4bad721b8ee020c27be3e7851L278
I don't think it's a big deal.